src/static/js/modules/util/web-storage-proxy.js

```diff
@@ -82,19 +91,24 @@ function setSessionType( sessionOnly ) {
  *   Use non-persistent storage (true) or persistent storage (false).
  * @returns {object} A local storage or session storage instance.
  */
-function _sessionOrLocal( sessionOnly ) {
+/* istanbul ignore next */
+function _storageType( storage ) {
```

Comment:
We're not really setting the storage type here, we're getting it, either as the passed value, the saved value from `L:19` or defaulting to `sessionStorage`. We actually set it above on `L82`